demo/test.c

```diff
@@ -2214,6 +2214,50 @@ static int test_s_mp_toom_sqr(void)
    return EXIT_FAILURE;
 }
 
+static int test_mp_pack_unpack(void)
+{
+   mp_int a, b;
+   int err;
+   size_t size, countp;
+   unsigned char *buf = NULL;
+
+   mp_order order = MP_LSB_FIRST;
+   mp_endian endianess = MP_NATIVE_ENDIAN;
+
+   if ((err = mp_init_multi(&a, &b, NULL)) != MP_OKAY)                       goto LTM_ERR;
+   if ((err = mp_rand(&a, 15)) != MP_OKAY)                                   goto LTM_ERR;
+
+   size = (size_t)(((mp_count_bits(&a) + 1) + 7) / 8);
+
+   buf = MP_MALLOC(size);
```

Comment:
mp_malloc or malloc? See below.

Reply:
I think `MP_MALLOC/MP_FREE` is better to make it more consistent with the rest.

Reply:
It should just be consistent in the test suite.

Reply:
But I think malloc makes more sense since we don't want to use a custom malloc in the test suite. This is different from the library code. I think the other tests also use plain malloc.